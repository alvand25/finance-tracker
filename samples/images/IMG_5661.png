version https://git-lfs.github.com/spec/v1
oid sha256:f025c2a340a643c689f07dd9625eebad631ec3edf1c3731f6400ea21b41ac674
size 13075705
