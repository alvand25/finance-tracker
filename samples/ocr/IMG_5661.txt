. \
/
ae 555
(ayy / ay
f 7
UE LO@ED)
, 51 IRE 5he ee
46-02 Queens B1vd.
: 5unnyside, HY 11104
C718) 706-6563 .
0,20 1b @ 6.99 /ib
4 WIT MOTHER EARTH ORG C 1.40 F '
CARROT BUNCH 2.69 F
i Regu1ar Price 2.99 , You 5aved 0,30
5CALLION5 0,99 F \
THAI KITE FI5H 5AU 5.09 F \
1.35 Ib @ 0.69 /1b
WT ONION5 5PANI5H 0.93 F
1.71 Ib @ 1.49 /Ib
WT YUKON POTATOE5 2,55 F
TAX 0.00
sant BALANCE 13.65
EMV CARD - C
KEY FOOD
46-02 QUEEN5 BLVD
5UNNY5IDE, NY 11104
payment Type: CHA5E VI5A
: Card: #x#A Hee HR Hgg55
payment At $13.65 Pur.
Cashback Aint: $0.00
futhorizati: *: oqgoap
5ccyencet: Fez
Jermiva1 1D