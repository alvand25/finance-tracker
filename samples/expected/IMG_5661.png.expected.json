{
    "store": "H_Mart",
    "items": [
        {
            "description": "ROTISSERIE CHICKEN",
            "price": 5.99
        },
        {
            "description": "JIN RAMEN MILD BOX",
            "price": 16.99
        },
        {
            "description": "SNOW ICE/COOKIE&CRM",
            "price": 8.99
        },
        {
            "description": "MINI CUCUMBERS",
            "price": 1.49
        },
        {
            "description": "MILK TEA LATTE",
            "price": 2.99
        },
        {
            "description": "CAFE AMERICANO",
            "price": 1.99
        },
        {
            "description": "MASAMAN CURRY PASTE",
            "price": 1.99
        },
        {
            "description": "BEEF GYOZA",
            "price": 5.99
        },
        {
            "description": "SAMANCO CHOCOLATE",
            "price": 4.99
        },
        {
            "description": "GREEN SEEDLESS GRAPES",
            "price": 5.56
        },
        {
            "description": "GROCERY",
            "price": 3.75
        },
        {
            "description": "PAPER BAG",
            "price": 0.05
        },
        {
            "description": "PAPER BAG",
            "price": 0.05
        }
    ],
    "subtotal": 60.82,
    "tax": 0.72,
    "total": 61.54,
    "payment_method": "VISA",
    "timestamp": "2024-04-13T15:20:00",
    "check_items_exactly": true
}